### PR TITLE
add 5HARE pool

### PR DESCRIPTION
**Pull request by 5pecia1**
Hello.
I'm pleased to be here.

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -3871,5 +3871,19 @@
...
+    "poolId": "73a527af03d38b3659f20b263000a1f72d3f3b867df5a2bb0e5516f1",
+    "social": {
+      "website": "https://5hare-pool.5pecia1.dev",
+      "twitter": "",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "pdpxpd",
+    "operatorName": "5pecia1",
```
Comment (Bez625) [inline registry.json:3886]: Please change this to be either your real full name, real first name or leave empty

File: registry.json [hunk 2]
```diff
@@ -3871,5 +3871,19 @@
...
+  },
+  {
+    "ticker": "5HARE",
+    "poolId": "73a527af03d38b3659f20b263000a1f72d3f3b867df5a2bb0e5516f1",
+    "social": {
+      "website": "https://5hare-pool.5pecia1.dev",
+      "twitter": "",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
```
Comment (Bez625) [inline registry.json:3883]: Please add an active social media presence

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json [hunk 3]
```diff
@@ -3898,5 +3898,19 @@
     "telegram": "lukas_psya",
     "operatorName": "",
     "github": "rafek47"
+  },
+  {
+    "ticker": "5HARE",
+    "poolId": "73a527af03d38b3659f20b263000a1f72d3f3b867df5a2bb0e5516f1",
+    "social": {
+      "website": "https://5hare-pool.5pecia1.dev",
+      "twitter": "https://twitter.com/5HARE_Pool",
```
Comment (Bez625) [inline registry.json:3907]: There is no activity on this account... We look for some amount of active social media presence. Also please can you make this just the twitter handle, e.g. `5HARE_Pool`

**Review by Bez625 — CHANGES_REQUESTED** [review]

**Comment by 5pecia1** [comment]
@Bez625  Okay. Thanks your review.
After activating Twitter, I will request a PR again.

**Pull request closed without merging**